routes/profileRoutes.js

```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+import Profile from '../controllers/profile';
+import { verifyToken, profileValidations } from '../middlewares';
+
+const {
```

Comment:
Whenever you are destructuring, It's a good practice to not step to th enext line until there are more than 3 destructured items.